src/ingest-pipeline/airflow/dags/codex_cytokit.py

```diff
@@ -488,17 +604,67 @@ def build_cwltool_cmd_sprm_to_anndata(**kwargs):
-     >> prepare_cwl_create_vis_symlink_archive >> t_build_cmd_create_vis_symlink_archive >> t_pipeline_exec_cwl_create_vis_symlink_archive >> t_maybe_keep_cwl_create_vis_symlink_archive
-     >> prepare_cwl_ome_tiff_pyramid >> t_build_cmd_ome_tiff_pyramid >> t_pipeline_exec_cwl_ome_tiff_pyramid >> t_maybe_keep_cwl_ome_tiff_pyramid
-     >> prepare_cwl_ome_tiff_offsets >> t_build_cmd_ome_tiff_offsets >> t_pipeline_exec_cwl_ome_tiff_offsets >> t_maybe_keep_cwl_ome_tiff_offsets
-     >> prepare_cwl_sprm_to_json >> t_build_cmd_sprm_to_json >> t_pipeline_exec_cwl_sprm_to_json >> t_maybe_keep_cwl_sprm_to_json
-     >> prepare_cwl_sprm_to_anndata >> t_build_cmd_sprm_to_anndata >> t_pipeline_exec_cwl_sprm_to_anndata >> t_maybe_keep_cwl_sprm_to_anndata
-     >> t_move_data >> t_expand_symlinks >> t_send_status >> t_join)
+    (
+            dag
+            >> t_log_info
+            >> t_create_tmpdir
+            >> t_send_create_dataset
+            >> t_set_dataset_processing
+
+            >> prepare_cwl_illumination_first_stitching
+            >> t_build_cwl_illumination_first_stitching
+            >> t_pipeline_exec_cwl_illumination_first_stitching
+            >> t_maybe_keep_cwl_illumination_first_stitching
+
+            >> prepare_cwl_cytokit
+            >> t_build_cwl_cytokit
+            >> t_pipeline_exec_cwl_cytokit
+            >> t_maybe_keep_cwl_cytokit
+
+            >> prepare_cwl_background_second_stitching
+            >> t_build_cwl_background_second_stitching
+            >> t_pipeline_exec_cwl_background_second_stitching
+            >> t_maybe_keep_cwl_background_second_stitching
+
+            >> prepare_cwl_sprm
+            >> t_build_cmd_sprm
+            >> t_pipeline_exec_cwl_sprm
+            >> t_maybe_keep_cwl_sprm
+
+            >> prepare_cwl_create_vis_symlink_archive
+            >> t_build_cmd_create_vis_symlink_archive
+            >> t_pipeline_exec_cwl_create_vis_symlink_archive
+            >> t_maybe_keep_cwl_create_vis_symlink_archive
+
+            >> prepare_cwl_ome_tiff_pyramid
+            >> t_build_cmd_ome_tiff_pyramid
+            >> t_pipeline_exec_cwl_ome_tiff_pyramid
+            >> t_maybe_keep_cwl_ome_tiff_pyramid
+
+            >> prepare_cwl_ome_tiff_offsets
+            >> t_build_cmd_ome_tiff_offsets
+            >> t_pipeline_exec_cwl_ome_tiff_offsets
+            >> t_maybe_keep_cwl_ome_tiff_offsets
+
+            >> prepare_cwl_sprm_to_json
+            >> t_build_cmd_sprm_to_json
+            >> t_pipeline_exec_cwl_sprm_to_json
+            >> t_maybe_keep_cwl_sprm_to_json
+
+            >> prepare_cwl_sprm_to_anndata
+            >> t_build_cmd_sprm_to_anndata
+            >> t_pipeline_exec_cwl_sprm_to_anndata
+            >> t_maybe_keep_cwl_sprm_to_anndata
+
+            >> t_move_data
+            >> t_expand_symlinks
+            >> t_send_status
+            >> t_join
+    )
+    t_pipeline_exec_cwl_illumination_first_stitching >> t_delete_internal_pipeline_files
```

Comment:
I'm not quite sure about this in particular -- my intention is that files which weren't previously exposed from the pipeline (only passed between steps) will be deleted from disk after the pipeline completes. I'm not sure if this is the right way to trigger that, though.

Reply:
That's interesting- I think that should work.  Let's give it a try.